examples/gcp-linux-tdx-vm/main.tf

```diff
@@ -0,0 +1,27 @@
+# --------------------------------------------------------
+#                     _       _       _
+#                    (_)     | |     | |
+#                     _ _ __ | |_ ___| |
+#                    | | '_ \| __/ _ \ |
+#                    | | | | | ||  __/ |
+#                    |_|_| |_|\__\___|_|
+# --------------------------------------------------------
+# Provision GCP virtual machine on Intel Xeon 4th Generation Scalable processors (code-named Sapphire Rapids) VM which supports Confidential Computing with Intel Trust Domain Extension (Intel TDX) technology
+# You will need to provide value of the variable project, which is your GCP project id when you do terraform apply
+module "linux_vm" {
+  #source              = "intel/gcp-vm/intel"
+  source              = "../../"
```

Comment:
remove the local source and put the github one back

Reply:
Done